spec/models/arbitration_rule_spec.rb

```diff
@@ -29,4 +29,14 @@
       expect(ArbitrationRule.get_by_rule_class('User')).to eq(user_rules)
     end
   end
+
+  describe '#arbitration_profile_id' do
+    it 'created with an arbitration_profile_id' do
+      profile = FactoryGirl.create(:arbitration_profile)
+
+      expect do
+        FactoryGirl.create(:arbitration_rule, :arbitration_profile_id => profile.id)
```

Comment:
For simplicity, this can also be written as:

```ruby
FactoryGirl.create(:arbitration_rule, :arbitration_profile => profile)
```

It sets the id, but also uses the object-level setter.